src/Contract/Time.purs

```diff
@@ -111,11 +109,12 @@ getCurrentEpoch = do
     $ BigInt.toString (bigInt :: BigInt.BigInt)
 
 -- | Get `EraSummaries` as used for Slot arithemetic.
--- | Details can be found https://ogmios.dev/api/ under "eraSummaries" query
+-- | Details can be found https://ogmios.dev/api/ under "eraSummaries" query.
```

Comment:
Is it fair to point to Ogmios docs here, given that the code is backend-agnostic?

Reply:
I think it is a good reference for the user, and Blockfrost response has an identical structure. 
Maybe we should specify a link to the Blockfrost endpoint spec as well?

Reply:
Ideally we should reference neither, because it is our own domain type that just happens to correspond to both Blockfrost's and Ogmios' ones, but since there would be no documentation otherwise, let's mention both, but form the message in a way that suggests that it is not a documentation for something we provide, but an external source.

E.g.:

More info can be found in Ogmios or Blockfrost docs (*links*), currently we use the same data type definition